stories/apps/launch/LoadingError.stories.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+import { withKnobs, boolean } from "@storybook/addon-knobs";
+
+import AppLaunchStoryBase from "./AppLaunchStoryBase";
+
+const errorObject = {
+    response: {
+        status: 404,
+        data: {
+            error_code: "ERR_NOT_FOUND",
+            reason: "Something went arwy",
```

Comment:
Minor tpyo?  `awry` 😄

Reply:
I uh... did that on purpose 👀 
It's even more arwy than awry 😆